File: 151003/Alekseeva/TaskREST/TaskREST/Models/Tweet.cs
using System.ComponentModel.DataAnnotations;

namespace TaskREST.Models;

public class Tweet
{
    public long Id { get; set; }
    public long CreatorId { get; set; }
    [MaxLength(32)] public string Title { get; set; } = string.Empty;
    [MaxLength(2048)] public string Content { get; set; } = string.Empty;
    public DateTime Created { get; init; }
    public DateTime Modified { get; set; }
}

File: 151001/Pastukhou/RV/src/main/java/by/bsuir/kirillpastukhou/impl/bean/Tweet.java
package by.bsuir.kirillpastukhou.impl.bean;

import lombok.Getter;
import lombok.Setter;

import java.sql.Date;

@Setter
@Getter
public class Tweet {
    private long id;
    private long creatorId;
    private String title;
    private String content;
    private Date created;
    private Date modified;
}
